templates/layout-slider.php

```diff
@@ -3,7 +3,11 @@
 
 $show_navigation = count($images) >= $nav_count_min;
 $slider_image_size = has_image_size('post-thumbnail') ? 'post-thumbnail' : 'large';
-$image_attr = Plugin::isLazyLoadActive() ? ['data-no-lazy' => '1'] : [];
+// Prevent wrong images height calculation caused by lazy loading.
+$image_attr = apply_filters('gallerya_lazyload_image_attributes', [
+  'data-no-lazy' => '1',
+  'class' => 'no-lazy attachment-large size-large',
```

Comment:
1. Why do we need the `attachment-large size-large` classes?

1. In case it is needed, why aren't we respecting `$slider_image_size`, which is being negotiated right above this line?

Reply:
@sun 
1. I saw that those classes where from the first image lost when injecting the `no-lazy` class as part of the `$image_attr`.
2. I'll correct that, thanks.